src/components/views/elements/Field.tsx

```diff
@@ -75,7 +77,7 @@ interface IProps {
 
 export interface IInputProps extends IProps, InputHTMLAttributes<HTMLInputElement> {
     // The ref pass through to the input
-    inputRef?: RefObject<HTMLInputElement>;
+    inputRef?: RefObject<HTMLInputElement> | RefCallback<HTMLInputElement>;
```

Comment:
Think this is the same as `Ref<HTMLInputElement>`

Reply:
same for the next 3 changes

Reply:
https://github.com/element-hq/element-web/pull/28452/commits/b8e54750a06ebd4bcbaf69236a2e8e9315565810